manifests/cf-manifest/deployments/050-logsearch.yml

```diff
@@ -0,0 +1,326 @@
+  env: (( grab meta.default_env ))
+  cloud_properties:
+    availability_zone: eu-west-1a
+    instance_type: m3.medium
+
+- name: kibana
+  network: cf1
+  stemcell: (( grab meta.stemcell ))
+  env: (( grab meta.default_env ))
+  cloud_properties:
+    availability_zone: eu-west-1a
+    instance_type: t2.micro
+
+- name: maintenance
+  network: cf1
+  stemcell: (( grab meta.stemcell ))
+  env: (( grab meta.default_env ))
+  cloud_properties:
+    availability_zone: eu-west-1a
+    instance_type: t2.micro
+
+- name: cluster_monitor
+  network: cf1
+  stemcell: (( grab meta.stemcell ))
+  env: (( grab meta.default_env ))
+  cloud_properties:
+    availability_zone: eu-west-1a
+    instance_type: m4.large
+
+- name: haproxy
+  network: cf1
+  stemcell: (( grab meta.stemcell ))
+  env: (( grab meta.default_env ))
+  cloud_properties:
+    availability_zone: eu-west-1a
+    instance_type: t2.micro
+
+jobs:
+- name: elasticsearch_master
+  release: logsearch
+  templates:
+  - {name: elasticsearch, release: logsearch}
+  resource_pool: elasticsearch_master
+  update:
+    serial: true
+  instances: 1
+  networks:
+  - name: cf1
+    static_ips: (( static_ips(0) ))
+  persistent_disk_pool: elasticsearch_master
+  properties:
+    elasticsearch:
+      node:
+        allow_master: true
+        allow_data: false
+  update:
+    serial: false
+    canaries: 1
+    max_in_flight: 1
+
+- name: haproxy
+  instances: 0
+  release: logsearch
+  templates:
+  - { name: haproxy, release: logsearch }
+  resource_pool: haproxy
+  networks:
+  - name: cf1
+  update:
+    serial: false
+    canaries: 1
+    max_in_flight: 1
+  properties:
+    haproxy:
+      backend_servers: (( grab jobs.ingestor.networks.[0].static_ips ))
+
+- name: cluster_monitor
+  instances: 0
+  release: logsearch
+  templates:
+  - { release: logsearch, name: queue }
+  - { release: logsearch, name: parser }
+  - { release: logsearch, name: ingestor_syslog }
+  - { release: logsearch, name: elasticsearch }
+  - { release: logsearch, name: elasticsearch_config }
+  - { release: logsearch, name: curator }
+  - { release: logsearch, name: kibana }
+  - { release: logsearch, name: nats_to_syslog }
+  resource_pool: cluster_monitor
+  networks:
+  - name: cf1
+  persistent_disk_pool: cluster_monitor
+  properties:
+    kibana:
+      elasticsearch: 127.0.0.1:9200
+      port: 5601
+    elasticsearch:
+      master_hosts: [127.0.0.1]
+      cluster_name: monitor
+      node:
+        allow_master: true
+        allow_data: true
+    redis:
+      host: 127.0.0.1
+    curator:
+      elasticsearch_host: 127.0.0.1
+      elasticsearch_port: 9200
+      purge_logs:
+        retention_period: 7
+    elasticsearch_config:
+      elasticsearch:
+        host: 127.0.0.1
+        port: 9200
+      bulk_data_files: []
+      templates:
+      - shards-and-replicas: "{ \"template\" : \"*\", \"order\" : 99, \"settings\" : { \"number_of_shards\" : 1, \"number_of_replicas\" : 0 } }"
```

Comment:
There are two levels of number of shards and replicas setting. First is global and defaults to 5 shards and one replica per shard. This seems to be the case here. Second is per index setting and you can override global value by specifying number of shards and replicas during index creation.